pkg/storage/node_liveness.go

```diff
@@ -394,7 +395,7 @@ func (nl *NodeLiveness) StartHeartbeat(
 					// Give the context a timeout approximately as long as the time we
 					// have left before our liveness entry expires.
 					ctx, cancel := context.WithTimeout(context.Background(), nl.livenessThreshold-nl.heartbeatInterval)
-					ctx, sp := ambient.AnnotateCtxWithSpan(ctx, "heartbeat")
+					ctx, sp := ambient.AnnotateCtxWithSpan(ctx, "liveness heartbeat loop")
```

Comment:
The Knife produces Heartbeats, suggest "the knife" as span name.

Reply:
Thanks for the suggestion! We'll put this idea into our backlog for when we finally get to enhancing our tracing to be completely unreadable to people who don't listen to synth-pop.